lib/Controller/BoardController.php

```diff
@@ -155,4 +155,15 @@ public function deleteAcl($aclId) {
 	public function clone($boardId) {
 		return $this->boardService->clone($boardId, $this->userId);
 	}
+
+	/**
+	 * @NoAdminRequired
+	 * @param $boardId
+	 * @param $owner
+	 * @param $newOwner
+	 * * @return null|void
+	 */
+	public function transferOwner($boardId, $owner, $newOwner) {
+		return $this->boardService->transferOwnership($owner, $newOwner);
```

Comment:
```suggestion
		return $this->boardService->transferBoardOwnership($boardId, $owner, $newOwner);
```

To make use of the newly added method to only transfer a single board.